docs/aws/audit/iamcompliance/rules/root_account_activity.mdx

```diff
@@ -23,6 +23,83 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all IAM users: To check the activity of the root account, you first need to list all IAM users. You can do this by running the command `aws iam list-users`. This will return a list of all IAM users in your AWS account.
+
+3. Check last used access key: To check when the root account was last used, you can use the command `aws iam get-access-key-last-used --access-key-id <access-key-id>`. Replace `<access-key-id>` with the access key ID of the root account. This will return the date and time when the root account was last used.
+
+4. Check CloudTrail logs: AWS CloudTrail logs all API calls made in your AWS account. You can use these logs to monitor the activity of the root account. To do this, you can use the command `aws cloudtrail lookup-events --lookup-attributes AttributeKey=Username,AttributeValue=root`. This will return a list of all API calls made by the root account.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary Python libraries: Before you can start writing the script, you need to install the necessary Python libraries. The AWS SDK for Python (Boto3) allows Python developers to write software that makes use of services like Amazon S3, Amazon EC2, etc. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Configure AWS Credentials: You need to configure your AWS credentials. You can configure credentials by using the AWS CLI or by directly adding them to your script. However, it's not a good practice to hard code your credentials into your script.
+
+```python
+import boto3
+
+aws_access_key_id = 'YOUR_ACCESS_KEY'
+aws_secret_access_key = 'YOUR_SECRET_KEY'
+aws_session_token = 'YOUR_SESSION_TOKEN'
+
+session = boto3.Session(
+    aws_access_key_id=aws_access_key_id,
+    aws_secret_access_key=aws_secret_access_key,
+    aws_session_token=aws_session_token,
+)
+```
+
+3. Write a Python script to check Root Account Activity: You can use the AWS CloudTrail service to monitor the root account activity. CloudTrail provides event history of your AWS account activity, including actions taken through the AWS Management Console, AWS SDKs, command line tools, and other AWS services.
+
+```python
+import boto3
+
+def check_root_account_activity():
+    client = boto3.client('cloudtrail')
+    response = client.lookup_events(
+        LookupAttributes=[
+            {
+                'AttributeKey': 'Username',
+                'AttributeValue': 'root'
+            },
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target="_blank">Click here for more details</a></b>